tModBuildTasks/OrganizeReferenceDestinations.cs

```diff
@@ -54,21 +54,16 @@ protected override void Run()
 				// Version is bugged in deps.json for ProjectReferences, doesn't reflect AssemblyVersion for whatever reason. Uses 1.0.0.
 				const string VersionHack = "1.0.0";
 
-				destinationSubDirectory = $"{BaseDirectory}/{assemblyName.Name}/{VersionHack}/";
+				destinationSubDirectory = Path.Combine(BaseDirectory, assemblyName.Name, VersionHack) + Path.DirectorySeparatorChar;
 			}
 			// Direct Managed References
 			else if (referenceSourceTarget == "ResolveAssemblyReference" && assemblyName != null) {
-				destinationSubDirectory = $"{BaseDirectory}/{assemblyName.Name}/{assemblyName.Version}/";
+				destinationSubDirectory = Path.Combine(BaseDirectory, assemblyName.Name, assemblyName.Version.ToString()) + Path.DirectorySeparatorChar;
 			}
 			// NuGet Packages
 			else if (!string.IsNullOrEmpty(nugetPackageId)) {
 				// This is used for all NuGet libraries, whether native or managed, whether rid-specific or agnostic.
-				if (!string.IsNullOrEmpty(pathInPackage)) {
-					destinationSubDirectory = $"{Path.GetDirectoryName($"{BaseDirectory}/{nugetPackageId}/{nugetPackageVersion}/{pathInPackage}").Replace('\\', '/')}/";
-				}
-				else {
-					destinationSubDirectory = $"{BaseDirectory}/{nugetPackageId}/{nugetPackageVersion}/";
-				}
+				destinationSubDirectory = Path.Combine(BaseDirectory, nugetPackageId, nugetPackageVersion, Path.GetDirectoryName(pathInPackage)) + Path.DirectorySeparatorChar;
```

Comment:
The need to add trailing slashes is a bit annoying, but understandable. Can you double check that it's required?

Reply:
Removing them broke everything.
![image](https://github.com/tModLoader/tModLoader/assets/6795251/d18a0475-f162-41e7-b9e7-0fd87d93087e)